js/components/selection-checkbox.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react'
+import pureRender from 'pure-render-decorator'
+import Answer from './answer'
+
+import '../../css/question.less'
+
+@pureRender
+export default class SelectionCheckbox extends Component {
+  constructor(props) {
+    super(props)
+  }
+
+
+  onChange(evt) {
+    const {questionKey, onSelectChange} = this.props
+    onSelectChange(questionKey, evt.target.checked)
+  }
+
+  render() {
+    const { selected, hideControls } = this.props
```

Comment:
Sorry for my nitpicking, but it would be great to have consistent whitespace pattern in one project (e.g. line 15 vs 20). It's the same about `variable=1` vs `variable = 1` or `[1,2,3]` vs `[1, 2, 3]` etc. I don't really care which ones we use, as long as we use the same pattern within the single project (although leaning towards the most common styles). :wink: